### PR TITLE
documentation: release notes for SMDDP 1.4.1 and SMDMP 1.9.0

### PR DESCRIPTION
**Pull request by mchoi8739**
Release notes for SMDDP 1.4.1 and SMDMP 1.9.0

*Testing done:*
- local build succeeded

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backword compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by roywei — APPROVED** [review]

File: doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst [hunk 1]
```diff
@@ -7,9 +7,44 @@ Release Notes
...
+**Migration to AWS Deep Learning Containers**
+
+This version passed benchmark testing and is migrated to the following AWS Deep Learning Containers (DLC):
+
+- PyTorch 1.11.0 DLC
+
+  .. code::
+
+    763104351884.dkr.ecr.<region>.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu113-ubuntu20.04-sagemaker
+    763104351884.dkr.ecr.<region>.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu115-ubuntu20.04-sagemaker
```
Comment (YangFei1990) [inline doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst:33]: I only see `cu113` one from the [release note](https://github.com/aws/deep-learning-containers/releases/tag/v1.0-pt-sagemaker-1.11.0-py38).

File: doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst [hunk 2]
```diff
@@ -5,9 +5,40 @@ Release Notes
...
+**Migration to AWS Deep Learning Containers**
+
+This version passed benchmark testing and is migrated to the following AWS Deep Learning Containers (DLC):
+
+- PyTorch 1.11.0 DLC
+
+  .. code::
+
+    763104351884.dkr.ecr.<region>.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu113-ubuntu20.04-sagemaker
+    763104351884.dkr.ecr.<region>.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu115-ubuntu20.04-sagemaker
```
Comment (YangFei1990) [inline doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst:26]: Same as smddp, only `cu113`

**Review by YangFei1990 — APPROVED** [review]

File: doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst (hunk 1 above)
Comment (mchoi8739) [reply doc/api/training/smd_data_parallel_release_notes/smd_data_parallel_change_log.rst]: fixed

File: doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst (hunk 2 above)
Comment (mchoi8739) [reply doc/api/training/smd_model_parallel_release_notes/smd_model_parallel_change_log.rst]: fixed

**Review by navinsoni — APPROVED** [review]

**Review by navinsoni — CHANGES_REQUESTED** [review]
can you please resolve conflicts

**Comment by mchoi8739** [comment]
> can you please resolve conflicts

Done!

**Review by shreyapandit — APPROVED** [review]

**Pull request merged**